multiversx_sdk_cli/cli_wallet.py

```diff
@@ -112,6 +112,7 @@ def wallet_new(args: Any):
 
     mnemonic = Mnemonic.generate()
     print(f"Mnemonic: {mnemonic.get_text()}")
+    print(f"Wallet address: {mnemonic.derive_key().generate_public_key().to_address(address_hrp).to_bech32()}")
```

Comment:
Hopefully, not a breaking change for some users :crossed_fingers: 

People who rely on a more structured output should use the `outfile` argument.